dynamics/client.py

```diff
@@ -173,17 +184,23 @@ def from_environment(cls):
         * DYNAMICS_CLIENT_ID: client id string
         * DYNAMICS_CLIENT_SECRET: client secret key string
         * DYNAMICS_SCOPE: comma separated list of urls
+        * DYNAMICS_RESOURCE: single target url
+
+        Only specify SCOPE or RESOURCE, but not both.
 
         :raises KeyError: An environment variable was not configured properly
         """
 
-        api_url = os.environ["DYNAMICS_API_URL"]
-        token_url = os.environ["DYNAMICS_TOKEN_URL"]
-        client_id = os.environ["DYNAMICS_CLIENT_ID"]
-        client_secret = os.environ["DYNAMICS_CLIENT_SECRET"]
-        scope = os.environ["DYNAMICS_SCOPE"].split(",")
+        api_url = os.environ.get("DYNAMICS_API_URL")
+        token_url = os.environ.get("DYNAMICS_TOKEN_URL")
+        client_id = os.environ.get("DYNAMICS_CLIENT_ID")
+        client_secret = os.environ.get("DYNAMICS_CLIENT_SECRET")
+        scope = os.environ.get("DYNAMICS_SCOPE")
+        if scope is not None:
+            scope = scope.split(",")
+        resource = os.environ.get("DYNAMICS_RESOURCE")
```

Comment:
This should be documented in the docs at [client constructor](https://github.com/MrThearMan/dynamics-client/blob/main/docs/docs.md#dynamicsclient) and [client from environment](https://github.com/MrThearMan/dynamics-client/blob/main/docs/docs.md#dynamicsclientfrom_environment)

Reply:
Done